features/steps/insights_sha_extractor.py

```diff
@@ -74,6 +74,7 @@ def start_sha_extractor(context, group_id=None):
     """Start SHA Extractor service."""
     if group_id:
         os.environ['CDP_GROUP_ID'] = group_id
+    os.environ['ALLOW_UNSAFE_LINKS'] = True
```

Comment:
what does it do?

Reply:
it allows to donwload from here https://github.com/RedHatInsights/insights-behavioral-spec/pull/317/files#diff-24bf4bd473ed35b9af4a0ad39287206996c0da6abd5d435a6ff4b376651e45eaR12 so I can use the mock

Reply:
how?

and how it is possible to use boolean as string?

Reply:
right! the fix is about to come